### ISSUE TITLE
Another undefined error....

### ISSUE THREAD

**Issue by anthony6608** [cowboysdude/MMM-NOAA3#35]
Hi,

First of thanks for putting this together for all of us! I spent a couple of hours trying to get it to work but to no avail. Initially I tried with accuweather until I saw a post of yours stating that dark sky is the best however both of them give the same results... everything is undefined. According to my dark sky account it isn't even hitting the API. Willing to try again if its a known issue. I also tried using the default config.

Thanks!

**Comment by cowboysdude** [comment]
Yes something is going on.....I just fired it up to test something and getting a ton of errors so I'm looking into them right now!! Thank you!

**Comment by anthony6608** [comment]
> Yes something is going on.....I just fired it up to test something and getting a ton of errors so I'm looking into them right now!! Thank you!

Cowboy, thanks for the quick reply... oddly enough I just tried again after taking a break and it works. 

I think my original issue was due to the fact that I copied the example config for accuweather and never changed it to dark sky. I just wanted to make sure and post and update for anyone else in the future who makes this stupid mistake! Thanks again man.
